main/tables.py

```diff
@@ -4,18 +4,40 @@
 from typing import ClassVar
 
 import django_tables2 as tables
+from django.db.models import QuerySet
 from django.utils.safestring import mark_safe
 
 from .models import Capacity, Funding, Project
+from .utils import order_queryset_by_property
 
 
 class ProjectTable(tables.Table):
     """Table for Project listing."""
 
     name = tables.Column(linkify=("main:project_detail", {"pk": tables.A("pk")}))
-    weeks_to_deadline = tables.Column(orderable=False)
-    total_effort = tables.Column(orderable=False)
-    days_left = tables.Column(orderable=False)
+
+    def order_weeks_to_deadline(
```

Comment:
All of these are those "magic" methods called by Django just because they have the right name, right?

Reply:
yes!